api/src/pdc/services/export/models/CarpoolRow.ts

```
@@ -28,7 +30,7 @@ export class CarpoolRow {
    * @param fields
    */
   public get(fields?: string[]): Partial<CarpoolRowData> {
-    return fields && fields.length ? pick(this.data, fields) : this.data;
+    return fields && fields.length ? pick(this.data, fields as any) : this.data;
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)